### ISSUE TITLE
Strange characters break src/view.php (still)

### ISSUE THREAD

**Issue by camilstaps** [clean-cloogle/cloogle#94]
For example, the icl of Data.Map contains a `ë` which breaks the python script.

**Comment by camilstaps** [comment]
URL: https://cloogle.org/src/view.php?lib=Platform#Data.Map;icl

Trace:

```
Traceback (most recent call last):
  File "cloogle_pygments.py", line 124, in 
    print(codecs.decode(outp, 'utf-8', 'ignore'))
UnicodeEncodeError: 'ascii' codec can't encode character '\xeb' in position 19811: ordinal not in range(128)
```

**Comment by dopefishh** [comment]
fixed by #98
